www/src/pages/languages.js

```diff
@@ -0,0 +1,102 @@
+/** @jsx jsx */
+import React from "react"
+import { jsx, useColorMode } from "theme-ui"
+import { Helmet } from "react-helmet"
+import styled from "@emotion/styled"
+
+import { langs, defaultLang } from "../utils/i18n"
+
+import Link from "../components/localized-link"
+import Container from "../components/container"
+import FooterLinks from "../components/shared/footer-links"
+import Breadcrumb from "../components/docs-breadcrumb"
+import LanguageThumb from "../components/languages/language-thumb"
+
+import { MdTranslate as TranslateIcon } from "react-icons/md"
+
+const TranslateBackground = styled(TranslateIcon)`
+  position: absolute;
+  right: calc(100% + 2rem);
+
+  font-size: 10rem;
+  opacity: 0.25;
+
+  color: ${p =>
+    p.colorMode === "dark" ? p.theme.colors.grey[10] : p.theme.colors.grey[90]};
+`
+
+const LanguagesContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  margin-left: -${p => p.theme.space[4]};
+  margin-bottom: ${p => p.theme.space[8]};
+`
+
+const allLanguages = [
+  { code: "en", name: "English", localName: "English" },
```

Comment:
should better all languages are in the i18n.json so that there is only one place where the languages are placed (and/or tranlsated)?